docs/spec/errors/error.json

```diff
@@ -55,26 +57,27 @@
         },
         "log": {
             "type": ["object", "null"],
+            "description": "Additional information added when logging the error.",
             "properties": {
                 "level": {
-                    "description": "The record severity.",
+                    "description": "The severity of the record.",
                     "type": ["string", "null"],
                     "default": "error",
                     "enum": ["debug", "info", "warning", "error", "fatal", null],
                     "maxLength": 1024
                 },
                 "logger_name": {
-                    "description": "The name of the logger which created the record.",
+                    "description": "The name of the used logger instance.",
                     "type": ["string", "null"],
                     "default": "default",
                     "maxLength": 1024
                 },
                 "message": {
-                    "description": "The exception's error message.",
+                    "description": "The additionally logged error message.",
```

Comment:
Not sure what you mean with "additionally" here. In addition to what?

Reply:
I basically copied over the current description from [fields.yml](https://github.com/elastic/apm-server/blob/master/processor/error/_meta/fields.yml#L84).

However, since it seems to be confusing how about:
description `log`:  `Information additionally logged to the original error`. 
description `log.message`: `Error message logged additionally to original error message.`

Reply:
The problem is that a `log` doesn't necessarily have an `exception`, and an `exception` does not necessarily have a `log`, so I don't think we should reference one from the other.

Reply:
Let's quickly discuss this tomorrow as it's relevant to the log/exception discussions we had today.